Documentation/deploy-master.md

```diff
@@ -68,7 +69,7 @@ ExecStartPre=/usr/bin/ln -sf /etc/flannel/options.env /run/flannel/options.env
 
 In order for flannel to manage the pod network in the cluster, Docker needs to be configured to use it. All we need to do is require that flanneld is running prior to Docker starting.
 
-*Note:* If the pod-network is being managed independently of flannel, this step can be skipped. See [kubernetes networking](kubernetes-networking.md) for more detail.
+*Note:* If the pod-network is being managed independently of flannel, or if Calico is being used, this step can be skipped. See [kubernetes networking](kubernetes-networking.md) for more detail.
```

Comment:
Maybe we just don't call out either tool and leave this to:

> If the pod-network is being managed independently, this step can be skipped.

If the admin is deploying their own pod-network (can be quite complex), this should be sufficiently clear. (/cc @joshix )